Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -141,11 +143,15 @@
 }
 
 WebExtension::WebExtension(NSDictionary *manifest, NSDictionary *resources)
-    : m_manifest([manifest copy])
-    , m_resources([resources mutableCopy])
-    , m_parsedManifest(true)
+    : m_resources([resources mutableCopy])
 {
     ASSERT(manifest);
+    ASSERT([NSJSONSerialization isValidJSONObject:manifest]);
+
+    NSData *manifestData = [NSJSONSerialization dataWithJSONObject:manifest options:0 error:nullptr];
+    ASSERT(manifestData);
```

Comment:
`RELEASE_ASSERT` since this will crash if `nil` when doing `setObject:` below. Might as well crash where the problem is instead.